### PR TITLE
feat: port over more developer tips from v2

### PR DESCRIPTION
**Pull request by paulbalaji**
resolves https://github.com/hyperlane-xyz/issues/issues/913

will rebase and merge after https://github.com/hyperlane-xyz/v3-docs/pull/41

preview at:

- [explorer debugging](https://hyp-v3-docs-git-feat-guides-dev-tips-abacus-works.vercel.app/docs/guides/developer-tips/explorer-debugging)
	- check out sub pages too
- [latencies](https://hyp-v3-docs-git-feat-guides-dev-tips-abacus-works.vercel.app/docs/guides/latencies)
- [faucets](https://hyp-v3-docs-git-feat-guides-dev-tips-abacus-works.vercel.app/docs/guides/token-faucets)

### CODE REVIEW

File: docs/guides/developer-tips/explorer-debugging.mdx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+Then paste a sender/recipient address or a transaction hash into the top search field. All matching messages will be shown in the result list. Click the row for more details.
+
+<figure>
+![](../../../static/img/explorer-debugging/explorer-search.png)
+<figcaption>Retrieving a transaction in the Hyperlane Explorer.</figcaption>
+</figure>
+
+## Debugging failed messages
+
+You'll know that a message failed to process because the bar at the top of the Message Explorer will turn red and the upper right section of the page will feature error reasons indicating why the message wasn't able to process.
```
Comment (jmrossy) [inline docs/guides/developer-tips/explorer-debugging.mdx:14]: This is no longer true, the message page was redesigned a while ago. Can you change to:

`The message status near the top of the page will show as "Error" and the upper right section of the page will specify a reason why the message wasn't able to be processed`

File: docs/guides/developer-tips/explorer-debugging/configuring-pi-chains.mdx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+![](../../../../static/img/explorer-debugging/explorer-add-custom-chain.png)
+
+### Chain Config Examples
+
+**A minimal chain config:**
+
+```js
+{
+  "chainId": 12345,
```
Comment (jmrossy) [inline docs/guides/developer-tips/explorer-debugging/configuring-pi-chains.mdx:23]: Out of date, here's a correct one:
```
  "chainId": 5,
  "name": "goerli",
  "protocol": "ethereum",
  "rpcUrls": [{ "http": "https://foobar.com" }],
```

File: docs/guides/developer-tips/explorer-debugging/configuring-pi-chains.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  "contracts": {
+    "mailbox": "0x12345..."
+  }
+}
+```
+
+**A chain config with a block explorer and block timings included:**
+
+```js
+{
```
Comment (jmrossy) [inline docs/guides/developer-tips/explorer-debugging/configuring-pi-chains.mdx:35]: ditto:
```
{
  "chainId": 5,
  "name": "goerli",
  "protocol": "ethereum",
  "rpcUrls": [{ "http": "https://foobar.com" }],
  "blockExplorers": [ {
      "name": "GoerliScan",
      "family": "etherscan",
      "url": "https://goerli.etherscan.io",
      "apiUrl": "https://api-goerli.etherscan.io/api",
      "apiKey": "12345"
  } ],
  "blocks": { "confirmations": 1, "estimateBlockTime": 13 },
  "mailbox": "0x123...",
  "interchainGasPaymaster": "0x123..."
}
```

File: docs/guides/developer-tips/explorer-debugging/rest-api.mdx [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+- `query`: address or hash to search (string)
+
+### APIs for Permissionless Interoperability chains&#x20;
+
+Hyperlane can be [permissionlessly deployed](../../../deploy-hyperlane.mdx) to any chain, but messages on PI chains cannot be identified by the default Hyperlane agents. To view details about messages from PI chains, query the `search-pi-messages` action. The search requires a chain config in the request body. Note, this same functionality is also available in the [explorer UI](../../../guides/developer-tips/explorer-debugging/configuring-pi-chains.mdx).
+
+```javascript
+const chainConfig = {
+  chainId: 1234,
```
Comment (jmrossy) [inline docs/guides/developer-tips/explorer-debugging/rest-api.mdx:116]: Ditto as above, we'll need an updated config schema here please

**Review by jmrossy — COMMENTED** [review]
Thanks for porting these over!! Just a few out of date things to fix

File: docs/guides/developer-tips/explorer-debugging.mdx (hunk 1 above)
Comment (paulbalaji) [reply docs/guides/developer-tips/explorer-debugging.mdx:14]: Ah yes thanks! made a mental note to remove but forgot to do it

**Review by jmrossy — APPROVED** [review]

**Pull request merged**